app/src/main/java/cmput301f17t13/com/catisadog/activities/LoginActivity.java

```diff
@@ -0,0 +1,355 @@
+package cmput301f17t13.com.catisadog.activities;
+
+import android.animation.Animator;
+import android.animation.AnimatorListenerAdapter;
+import android.annotation.TargetApi;
+import android.content.pm.PackageManager;
+import android.support.annotation.NonNull;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.app.LoaderManager.LoaderCallbacks;
+
+import android.content.CursorLoader;
+import android.content.Loader;
+import android.database.Cursor;
+import android.net.Uri;
+import android.os.AsyncTask;
+
+import android.os.Build;
+import android.os.Bundle;
+import android.provider.ContactsContract;
+import android.text.TextUtils;
+import android.view.KeyEvent;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.view.inputmethod.EditorInfo;
+import android.widget.ArrayAdapter;
+import android.widget.AutoCompleteTextView;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import org.joda.time.DateTime;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+import cmput301f17t13.com.catisadog.R;
+
+import static android.Manifest.permission.READ_CONTACTS;
+
+/**
+ * A login screen that offers login via email/password.
+ */
+public class LoginActivity extends AppCompatActivity implements LoaderCallbacks<Cursor> {
+
+    /**
+     * Id to identity READ_CONTACTS permission request.
+     */
+    private static final int REQUEST_READ_CONTACTS = 0;
+
+    /**
+     * A dummy authentication store containing known user names and passwords.
+     * TODO: remove after connecting to a real authentication system.
```

Comment:
#16
Whenever making a `TODO` item in the code, please create a GitHub issue and reference it. For example:
```
TODO(#16): Create issue for this TODO and reference the issue #
```

Reply:
I didn't write any code in this file. I dumped in the default Android Studio Login Activity... we can delete it to a vanilla version if you want.

Reply:
Up to you. If you keep it in please reference an issue (possibly related to a larger goal such as authentication)
.